packages/valory/skills/dynamic_nft_abci/handlers.py

```diff
@@ -69,6 +69,14 @@
 AVERAGE_PERIOD_SECONDS = 10
 DISCORD_ID_REGEX = r"^\d{16,20}$"
 
+BADGE_LEVELS = {
+    "Idle": 100,
+    "Basic": 50000,
+    "Legendary": 100000,
+    "Epic": 150000,
+    "Super Epic": None,
```

Comment:
Because there's no upper limit on it (it's the highest level). Then we do (see the None check):
```
for level, threshold in BADGE_LEVELS.items():
    if not threshold or user_points < threshold:
        user_level = level
```

Reply:
> Is this as this is our previously minted nfts?
These limits can be changed as we like as only affect to our (dynamic) metadata. Minted tokens do not contain any references to thresholds.

Reply:
Perfect thanks